modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterTag.java

```diff
@@ -45,7 +45,22 @@ public interface ClusterTag extends NetworkMessage, Serializable {
      * @param name Cluster name.
      * @return Cluster tag instance.
      */
-    static ClusterTag clusterTag(CmgMessagesFactory msgFactory, String name) {
+    static ClusterTag randomClusterTag(CmgMessagesFactory msgFactory, String name) {
```

Comment:
I assume random tag should be used only in tests, let's annotate it as TestOnly? 

If it's really only for tests, do we really need a separate method? Name is changed, so we have to change all test classes in any case))

Reply:
I added the annotation.

I think it still makes sense to use a special method, just to reduce duplication